src/agent/agent_info/tests/CMakeLists.txt

```diff
@@ -2,12 +2,16 @@ find_package(GTest CONFIG REQUIRED)
 
 add_executable(agent_info_test agent_info_test.cpp)
 configure_target(agent_info_test)
-target_include_directories(agent_info_test PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/../src)
-target_link_libraries(agent_info_test PRIVATE AgentInfo Persistence GTest::gtest)
+target_include_directories(agent_info_test PRIVATE
+    ${CMAKE_CURRENT_SOURCE_DIR}/../src
+    ${CMAKE_CURRENT_SOURCE_DIR}/../../persistence/tests/mocks)
```

Comment:
Note for the future: we need to do this also with tests that use the http client mock.